### DISCUSSION TITLE
Erro license

### DISCUSSION THREAD

**Discussion by robsonmgalvao** [aclap-dev/video-downloadhelper#1307] [Help and Q&A]
### Your question

Hello good afternoon! I just bought the license to use the VideoDownloadHelper extension and paid 18.50EU and when I paste the key it gives this message: License locked (check your email) But I haven't received an email yet. I need to use the tool.

### Addon & CoApp version and details
Video DownloadHelper
my email robson.marques@outlook.com

**Comment by jcfrog** [comment]
Hi,
Sorry, you should have received an email to reset your counter. Maybe in your spam box.
Anyway, I have reset your installation counter, should be OK now.
